python/CMakeLists.txt

```diff
@@ -1,38 +1,34 @@
-set(Python_ADDITIONAL_VERSIONS 3.7 3.6 3.5)  # adjust with CMake minimum FindPythonInterp
-find_package(PythonLibsNew 2.7 REQUIRED)
 find_package(SWIG REQUIRED)
+INCLUDE(UseSWIG)
```

Comment:
Nit-pick: @bast and I prefer CMake commands in lowercase, _e.g._ `include(UseSWIG)`